### ISSUE TITLE
summary.html not found although datasetDesc.conf was used

### ISSUE THREAD

**Issue by matthewspeir** [maximilianh/cellBrowser#109]
I'm seeing the error message 'summary.html was not found' even though I created a datasetDesc.conf for my datasets.

This is visible on https://hgwdev.gi.ucsc.edu/~mspeir/cb_demo/. On the first load of the page, it might look like everything is fine, but if you switch between the datasets, you should see the error. What's interesting is that when I click the 'Info' button the description loads fine, it's just if I click the 'Open' button.

Not really sure what's going on here. 

Data for the quake-gbm dataset, which is part of this browser, is here: /hive/users/mspeir/cellbrowserTest/quakeGlio_GSE84465/quakeGlioblastomaGSE84465. I can list the others if you want.

**Comment by maximilianh** [comment]
This is a bug that I fixed recently but I still haven't pushed this version
sorry...

**Comment by maximilianh** [comment]
Finally pushed a new release, 0.5.22 should have this fixed.

I couldn't update your version as I don't have permissions here:

cbUpgrade -o ~mspeir/cb_demo/

**Comment by matthewspeir** [comment]
Hmm, tried updating that directory with cbUpgrade and I got the following error:
```
$ cbUpgrade -o ~/public_html/cb_demo/
INFO:root:Copying js, css and img files to /cluster/home/mspeir/public_html/cb_demo/
Traceback (most recent call last):
  File "/cluster/home/mspeir/miniconda3/bin/cbUpgrade", line 10, in <module>
    sys.exit(cbUpgradeCli())
  File "/cluster/home/mspeir/miniconda3/lib/python3.6/site-packages/cellbrowser/cellbrowser.py", line 3889, in cbUpgradeCli
    cbUpgrade(outDir, devMode=options.devMode)
  File "/cluster/home/mspeir/miniconda3/lib/python3.6/site-packages/cellbrowser/cellbrowser.py", line 3867, in cbUpgrade
    datasets = findDatasets(outDir)
  File "/cluster/home/mspeir/miniconda3/lib/python3.6/site-packages/cellbrowser/cellbrowser.py", line 3567, in findDatasets
    inStr = inStr.decode("utf8")
AttributeError: 'str' object has no attribute 'decode'
```

Running cbBuild also gives the same error.

**Comment by maximilianh** [comment]
I just broke it, yes the same happened to me, sorry, just loooking...

**Comment by maximilianh** [comment]
Should be fixed with 0.5.25, though still looking trying to understand what
happened.

**Comment by matthewspeir** [comment]
This is looking good now. Thanks, Max.
